### PR TITLE
added remaining list count number at show more #6731

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

**Comment by gautampanday** [comment]
show-more-list

File: templates/webshop/search-results.jinja [hunk 1]
```diff
@@ -34,16 +34,15 @@
...
-                      <span class="glyphicon glyphicon-plus"></span>
-                    </span>
-                    <span class="show-on-more">
-                      {% trans %}Show Less{% endtrans %}
-                      <span class="glyphicon glyphicon-minus"></span>
-                    </span>
-                  </div>
+                <div class="more btn btn-default btn-sm toggle-show-more">
+                  <span class="hide-on-more">
+                    {% set remaining_list = 5 %}
```
Comment (tarunbhardwaj) [inline templates/webshop/search-results.jinja]: move this line to start and use this variable in whole file. call this variable `facet_show_limit`

File: templates/webshop/search-results.jinja [hunk 2]
```diff
@@ -34,16 +34,15 @@
...
+                    <span class="glyphicon glyphicon-plus">{{ value['terms']|length-remaining_list }}</span> {% trans %}More{% endtrans %}
```
Comment (tarunbhardwaj) [inline templates/webshop/search-results.jinja]:
```jinja
{{ value['terms'] | length - remaining_list }}
```

Comment (tarunbhardwaj) [inline templates/webshop/search-results.jinja]: use normal + sign rather that icon.

**Pull request merged**